src/embed/liveboard.ts

```diff
@@ -395,11 +313,12 @@ export class LiveboardEmbed extends V1Embed {
                 logger.warn('Full height is currently only supported for Liveboard embeds.' +
                     'Using full height with vizId might lead to unexpected behavior.');
             }
-
-            this.on(EmbedEvent.RouteChange, this.setIframeHeightForNonEmbedLiveboard);
-            this.on(EmbedEvent.EmbedHeight, this.updateIFrameHeight);
-            this.on(EmbedEvent.EmbedIframeCenter, this.embedIframeCenter);
-            this.on(EmbedEvent.RequestVisibleEmbedCoordinates, this.requestVisibleEmbedCoordinatesHandler);
+            this.fullHeightClient = new FullHeight({
+                getIframe: () => this.iFrame,
```

Comment:
Maybe just pass the instance of liveboardEmbed here, and create these constructs inside the constructor? These constructs are very specific to the embed instance anyway, and this parts seems a little duplicated.